build-linux/howto.md

```diff
@@ -10,16 +10,20 @@ To build Linux packages, you will need GPG v1 keys to sign the packages.  Do the
 
 ```
 brew install gpg1
-mkdir build-linux/gnupg
-chmod 700 build-linux/gnupg
-gpg1 --homedir=build-linux/gnupg --gen-key
+mkdir build-linux/gpg-deb
```

Comment:
reminder for us: this change will work for the deb package but not the rpm one i think

Reply:
working that out now